tests/integration/conftest.py

```diff
@@ -36,6 +39,65 @@ def session(request, api_url):
         yield s
 
 
+# TODO combine with session in some way?
+@pytest.fixture(scope='module',
+                params=[['admin', 'letmein']],
+                ids=['admin'])
+def girder_client(request, api_url):
+    username, password = request.param
+    client = GirderClient(apiUrl=api_url)
+    client.authenticate(username, password)
+
+    yield client
+
+
+@pytest.fixture(scope='module')
+def test_file(tmpdir_factory):
+    path = tmpdir_factory.mktemp('test').join('test.txt')
+    # Three times our read buffer for docker tests
+    path.write('abc'*65536)
```

Comment:
It's probably prudent to have our test file not be an exact multiple of the buffer length. (Or, we should at least cover that case.)

Reply:
I added a few more bytes

Reply:
I added a few more bytes